pkg/utils/utils.go

```diff
@@ -32,6 +32,7 @@ const (
 	sysBusPciDrivers      = "/sys/bus/pci/drivers"
 	sysBusPciDriversProbe = "/sys/bus/pci/drivers_probe"
 	sysClassNet           = "/sys/class/net"
+	sysKernelCmdLine      = "/proc/cmdline"
```

Comment:
I don't like the sys in the name of the variable. It's not `/sys`

Reply:
My mistake, copy paste issue =) .
I will rename it to procKernelCmdLine